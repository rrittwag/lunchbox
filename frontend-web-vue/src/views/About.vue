<template>
  <div class="sm:p-4">
    <h1
      class="
        flex
        items-center
        justify-center
        sm:justify-start sm:h-16
        px-4
        text-2xl text-neutral-800
      "
    >
      Info
    </h1>
    <address>
      <a href="https://github.com/data-experts/Lunchbox">Github</a>
    </address>
  </div>
</template>
