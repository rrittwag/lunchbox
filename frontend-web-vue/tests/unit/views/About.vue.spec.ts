import About from '@/views/About.vue'
import { shallowMount } from '@vue/test-utils'

describe('About', () => {
  test('renders snapshot', () => {
    const wrapper = shallowMount(About)
    expect(wrapper.element).toMatchSnapshot()
  })
})
