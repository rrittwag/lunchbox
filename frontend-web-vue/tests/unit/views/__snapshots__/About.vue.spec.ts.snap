// Vitest Snapshot v1

exports[`About > renders snapshot 1`] = `
<div
  class="sm:p-4"
>
  <h1
    class="flex items-center justify-center px-4 text-2xl text-neutral-800 sm:h-16 sm:justify-start"
  >
    Info
  </h1>
  <address>
    <a
      href="https://github.com/data-experts/Lunchbox"
    >
      Github
    </a>
  </address>
</div>
`;
